src/environments/environment.development.ts

```diff
@@ -1,4 +1,7 @@
 export const environment = {
     consumerBaseUrl: 'http://localhost:9093',
     producerBaseUrl : 'http://localhost:9000',
+    socketUrl : 'http://localhost:4000',
```

Comment:
I think websockets are supposed to use the 'wss' protocol instead of http
```suggestion
    socketUrl : 'wss://localhost:4000',
```